game/graphics/renderer.cpp

```diff
@@ -84,8 +84,22 @@ void Renderer::resetSwapchain() {
 
   if(settings.fxaaEnabled) {
     fxaa.sceneTonemapped = device.attachment(TextureFormat::RGBA8, w, h);
+    } else if(settings.cmaa2Enabled) {
+    cmaa2.sceneTonemappedUav = device.image2d(TextureFormat::RGBA8, w, h);
```

Comment:
Better not to use `UAV` sufix here, as it is DX12 terminology and engine largely follows Vulkan terms